Update

Knock-Out opened the pull request:
Update doc.

Knock-Out committed:
Update data-evento.php

Inverti data da:

Anno
Giorno
Mese

a

Giorno
Mese
Anno

Knock-Out committed:
Update custom-field-articoli.md

#16 #42

Knock-Out committed:
Update footer.md

data-element="legal-notes-body"

pcrt reviewed (commented):
io lascerei come di defualt perchè rispetta il layout html del modello scuole

pcrt reviewed (changes requested):
lascialo come di default e rifai il commit

Knock-Out committed:
Update data-evento.php

Correzione data evento

Knock-Out committed:
Card Modalità di accesso

Layout di tipo Card per il campo gruppi Luoghi.

Istruzioni
Creare un nuovo campo e chiamarlo "Modalità di accesso"

All’interno della tab Generale impostare i seguenti parametri:

Tipo: Editor (editor)
Nome: modalita-di-accesso
Etichetta: Modalità di accesso
Obbligatorio: Sì
Mostra pulsanti: Nascondi tutti i pulsanti
Filtro: Usa impostazioni Plugin
Stato: Pubblicato
Gruppo di campi: Luoghi
Categoria: I luoghi

Nella tab Opzioni impostare:

Modificabile in: entrambi
Mostra etichetta: mostra
Visualizza il campo: dopo il contenuto del display
Layout: card-modalita-accesso
Mostra se di sola lettura: eredita

Knock-Out committed:
Update custom-field-articoli.md

commit 9b1f9b6
